Data/DataModel/Event.cs

```diff
@@ -312,5 +312,11 @@ public bool AreStandingsAvailableNow
         /// True if Blazor can run on pages for this event. Should normally be true, but can be disabled in an emergency
         /// </summary>
         public bool AllowBlazor { get; set; }
+
+        /// <summary>
+        /// Short-term hacks for modifying behavior without adding new properties all the time.
+        /// Note: If a hack lasts more than a few months, it should probably be promoted to a real property.
+        /// </summary>
+        public string EphemeralHacks { get; set; }

```

Comment:
Consider adding ```[NotMapped]``` properties with getters that encapsulate the string checking logic for the strings you're adding so we don't have to spread the logic around and so we have central-ish knowledge of what they are

Reply:
Good idea, will do!